### PR TITLE
The Japanese translation is now complete.

### PR DESCRIPTION
**Pull request by Mr-IjIj**
This time, we fixed the changes in the src/ directory before and translated the middleware documentation.

Sorry for the slight delay.
Please check it out.

### CODE REVIEW

**Comment by Mr-IjIj** [comment]
The name of the brunch has changed, but the original brunch has gone wrong, so please forgive me.

File: docs/ja/authenticators.rst [hunk 1]
```diff
@@ -192,173 +170,154 @@ configuring your app as follows::
...
+-  **cookie**: クッキーオプションの配列:
 
-   -  **name**: Cookie name, default is ``CookieAuth``
-   -  **expire**: Expiration, default is ``null``
-   -  **path**: Path, default is ``/``
-   -  **domain**: Domain, default is an empty string \`\`
-   -  **secure**: Bool, default is ``false``
-   -  **httpOnly**: Bool, default is ``false``
-   -  **value**: Value, default is an empty string \`\`
+-  **name**: クッキー名, デフォルトは ``CookieAuth``
```
Comment (markstory) [inline docs/ja/authenticators.rst:213]: This item needs to be indented further.

**Review by markstory — COMMENTED** [review]
Looks good. But there is a conflict that needs to be resolved before this can be merged.

File: docs/ja/authenticators.rst (hunk 1 above)
Comment (Mr-IjIj) [reply docs/ja/authenticators.rst]: Sorry. Sorry for lack of confirmation. ....
Please check under the request to push the one you just fixed.
Thank you for finding the mistake.
Sorry for the late reply.

**Review by Mr-IjIj — COMMENTED** [review]
Fixed the indentation.
Please check.

**Comment by markstory** [comment]
Looks good. The conflicts with master will need to be resolved before I can merge this though.

**Comment by Mr-IjIj** [comment]
Conflicts resolved.
Please check.
Sorry for repeating. Thank you again.
I feel like this was a very valuable experience!

**Comment by markstory** [comment]
Thank you @Mr-IjIj I'll get the docs deployed later today.

**Pull request merged**